src/ansys_sphinx_theme/__init__.py

```diff
@@ -31,6 +30,14 @@ def get_html_theme_path():
     return Path(__file__).parents[0].absolute()
 
 
+def get_version_match(semver):
```

Comment:
I really like you included this function here. This should be available by default to all projects using the template and allows us to forget about adding it manually. Thanks @Revathyvenugopal162.